File: 04 Technical/01 Client/01 Logics/01 Javascript/08 Fibonacci Series.js
// Fibonacci Series

let given_number = 10;

let fibonacci_sequence = [0, 1];

for (let i = 0; i <= given_number - 3; i++) {
	add_number = fibonacci_sequence[i] + fibonacci_sequence[i + 1];
	fibonacci_sequence.push(add_number);
}

console.log(fibonacci_sequence);

File: 04 Technical/02 Server/01 Logics/01 Python/01 General/08 Fibonacci Series.py
# Fibonacci Series

given_number = 10  # int(input())
print(given_number)

print("--------------")

fibonacci_sequence = [0, 1]

for counter in range(0, given_number-2):
    add_number = fibonacci_sequence[counter] + fibonacci_sequence[counter+1]
    fibonacci_sequence.append(add_number)

print(fibonacci_sequence)

print("=====================")



